docs/quickstart.rst

```diff
@@ -20,42 +20,60 @@ Activate the `virtual environment <https://docs.python.org/3/tutorial/venv.html>
 
 	$ python -m pip install -e .
 
-Either installation method will install all of the dependencies which are required to run Scenic.
 If you will be developing Scenic, you will want to use a variant of this command: see :doc:`developing`.
 
+Either installation method will install all of the dependencies which are required to run Scenic, with the exception of `Blender <https://www.blender.org/>`_ and `OpenSCAD <https://openscad.org/>`_ which must also be installed on the system. This is best done by going to the download pages for `Blender <https://www.blender.org/download/>`__ and `OpenSCAD <https://openscad.org/downloads.html>`__.
+
 .. note::
 
-	If you are using Windows, or encounter any errors during installation or trying the examples below, please see our :doc:`install_notes` for suggestions.
+	If you are using Windows or an Apple silicon machine, or encounter any errors during installation or trying the examples below, please see our :doc:`install_notes` for suggestions.
 
 Trying Some Examples
 --------------------
 
 The Scenic repository contains many example scenarios, found in the :file:`examples` directory.
 They are organized by the simulator they are written for, e.g. :abbr:`GTA (Grand Theft Auto V)` or Webots; there are also cross-platform scenarios written for Scenic's abstract application domains, e.g. the :ref:`driving domain <driving_domain>`.
```

Comment:
This is no longer true, with the `3d` and `cav23_figures` folders. If those scenarios are all for visualization purposes only and not suitable for simulation, maybe you can put them under a `3d_demos` folder or something more descriptive (not just `3d` since there will be 3D scenarios in the other folders too).

 In any case, if the folder is no longer going to be simply organized by simulator we should probably add a `README.md` with a bullet-point list briefly explaining what can be found in each subfolder.